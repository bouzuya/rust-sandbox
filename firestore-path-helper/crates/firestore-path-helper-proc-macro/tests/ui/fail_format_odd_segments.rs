fn main() {
    firestore_path_helper_proc_macro::firestore_path_helper!(
        "users/{user_id}/repos",
        user_id = String
    );
}
