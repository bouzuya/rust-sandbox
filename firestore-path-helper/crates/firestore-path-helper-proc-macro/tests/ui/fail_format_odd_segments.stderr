error: format must contain an even number of segments separated by '/'
 --> tests/ui/fail_format_odd_segments.rs:3:9
  |
3 |         "users/{user_id}/repos",
  |         ^^^^^^^^^^^^^^^^^^^^^^^
